packages/utils/src/deepClone.ts

```diff
@@ -0,0 +1,15 @@
+export const deepClone = <T>(value: T): T => {
```

Comment:
There are ~ 12 more places where `JSON.parse(JSON.stringify...` is used, I guess it would be nice to use this util on all those places?

Reply:
da918636c94c0c0cfc1d158e15332ee83054edba